pkg/controller/registry/resolver/resolver.go

```diff
@@ -112,7 +111,6 @@ func (r *OperatorsV1alpha1Resolver) ResolveSteps(namespace string, sourceQuerier
 		if subExists && existingSubscription.Status.CurrentCSV != op.Identifier() {
 			existingSubscription.Status.CurrentCSV = op.Identifier()
 			updatedSubs = append(updatedSubs, existingSubscription)
-			continue
```

Comment:
Why are these removed?

Reply:
I didn't (and doesn't) look like this `continue` is skipping any logic between here and the next iteration of the loop.